src/main/java/com/addthis/meshy/MeshyServer.java

```diff
@@ -384,6 +389,25 @@ public void connectPeer(InetSocketAddress address) {
         }
     }
 
+    public void connectPeers(List<InetSocketAddress> addresses) {
+        connectPeers(addresses, meshyUnitDelay, meshyMaxDelayUnit, meshyMaxRetries);
+    }
+
+    public void connectPeers(List<InetSocketAddress> addresses, int unitDelay, int maxDelayUnit, int maxRetries) {
+        Backoff backoff = new Backoff(unitDelay, maxDelayUnit);
+        int retries=0;
+        while(getChannelCount()==0 && retries++<maxRetries) {
+            if (retries > 1) {
+                log.info("gagagaga mss will retry in: " + backoff.calcDelay() + " milliseconds.");
```

Comment:
Let's make sure we remove the gagagaga before merging; also say what is being retried (the connection to seeds)